### DISCUSSION TITLE
How do I manually make webRTC status disconnected?

### DISCUSSION THREAD

**Discussion by ghost** [stasel/WebRTC-iOS#79] [Q&A]

**Comment by stasel** [comment]
You can terminate the connection by calling `peerConnection.close()`

**Reply by ghost** [reply]
I have tried it. However, it completely closed the connection and was unable to reconnect when it came back to main. Any advice on this?

**Reply by stasel** [reply]
Yeah that true, closing the peer connection is not reversible, if you close a connection you need to recreate a new instance of the peer connection and exchange all the information once again.

If you want to pause the transmission I suggest trying to enable/disable the video/audio transceivers
